packages/application-components/src/components/dialogs/info-dialog/info-dialog.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CloseIcon,
+  Constraints,
+  SecondaryIconButton,
+  Spacings,
+  Text,
+} from '@commercetools-frontend/ui-kit';
+import ModalContainer from '../modal-container';
+import styles from './info-dialog.mod.css';
+
+// Like "Spacings.Inline", but with the `justify-content: space-between` style
+const Stretched = props => (
```

Comment:
Any plans on having this in UI Kit at some point? I can verify that we will remove A LOT of custom styles in the MC 🤓 cc/ @montezume

Reply:
@qmateub you can has an issue create 🐕

Reply:
I guess we can also extend the `Spacings.Inline` component to support this mode